src/layouting/data_limits.jl

```diff
@@ -209,6 +209,19 @@ function data_limits(plot::Surface)
     return Rect3f(mini, maxi .- mini)
 end
 
+function data_limits(plot::Mesh)
+    xyz = plot.mesh[].position
+    mini, maxi = bbox_mesh(xyz)
+    if length(mini) == 2
```

Comment:
Is there some nicer way to do this dimension agnostic?

Reply:
Typically these cases use `to_ndim(Vec3f, mini, 0f0)`

Reply:
sweet, thanks!